README.md

```diff
@@ -1,159 +1,173 @@
+The easiest way to communicate with the team is via GitHub issues.
 
-Please help us keep this repository clean, inclusive, and fun\! We will not tolerate any abusive, rude, disrespectful or inappropriate behavior. Read our [Code of Conduct](https://opensource.microsoft.com/codeofconduct/) for more details.
+Please file new issues, feature requests and suggestions, but **DO search for similar open/closed pre-existing issues before creating a new issue.
 
 If you would like to ask a question that you feel doesn't warrant an issue (yet), please reach out to us via Twitter:
 
-  * Rich Turner, Program Manager: [@richturn\_ms](https://twitter.com/richturn_ms)
-
-  * Dustin Howett, Engineering Lead: [@dhowett](https://twitter.com/DHowett)
-  
-  * Michael Niksa, Senior Developer: [@michaelniksa](https://twitter.com/MichaelNiksa)
-
-  * Kayla Cinnamon, Program Manager (especially for UX issues): [@cinnamon\_msft](https://twitter.com/cinnamon_msft)
+* Kayla Cinnamon, Program Manager: [@cinnamon\_msft](https://twitter.com/cinnamon_msft)
+* Rich Turner, Program Manager: [@richturn\_ms](https://twitter.com/richturn_ms)
+* Dustin Howett, Engineering Lead: [@dhowett](https://twitter.com/DHowett)
+* Michael Niksa, Senior Developer: [@michaelniksa](https://twitter.com/MichaelNiksa)
 
-# Developer Guidance
+## Developer Guidance
 
 ## Build Prerequisites
 
-* You must be running Windows 1903 (build >= 10.0.18362.0) or above in order to run Windows Terminal.
-* You must have the [1903 SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk) (build 10.0.18362.0) installed.
-* You must have at least [VS 2019](https://visualstudio.microsoft.com/downloads/) installed.
-* You must install the following Workloads via the VS Installer. Opening the solution will [prompt you to install missing components automatically](https://devblogs.microsoft.com/setup/configure-visual-studio-across-your-organization-with-vsconfig/).
-  - Desktop Development with C++
-  - Universal Windows Platform Development
-  - **The following Individual Components**
-     - C++ (v142) Universal Windows Platform Tools
-
-* You must also [enable Developer Mode in the Windows Settings app](https://docs.microsoft.com/en-us/windows/uwp/get-started/enable-your-device-for-development) to locally install and run the Terminal app.
-
+* You must be running Windows 1903 (build >= 10.0.18362.0) or later in order to run Windows Terminal
+* You must [enable Developer Mode in the Windows Settings app](https://docs.microsoft.com/en-us/windows/uwp/get-started/enable-your-device-for-development) to locally install and run Windows Terminal
+* You must have the [Windows 10 1903 SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk) installed.
+* You must have at least [VS 2019](https://visualstudio.microsoft.com/downloads/) installed
```

Comment:
```suggestion
* You must have at least [VS 2019](https://visualstudio.microsoft.com/downloads/) installed.
```

Reply:
Bullet points should not end in a full-stop/period